addons/website_event_meet/views/event_meet_templates_list.xml

```diff
@@ -119,13 +119,24 @@
         t-att-data-toggle="meeting_room_data_toggle"
         t-att-data-target="meeting_room_data_target">
         <div class="text-decoration-none w-100 h-100 p-3">
-            <div class="o_wevent_meeting_room_corner_ribbon" t-if="meeting_room.room_is_full">Full</div>
```

Comment:
Change about the "room full" ribbon should also be mentioned in the commit message, so we understand why you did it.

Reply:
I modified the commits to preserve the ribbon.